support-figma/auto-content-preview-widget/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "list-preview-widget",
-  "version": "1.1.0",
+  "version": "1.0.1",
```

Comment:
Was this version change intentional?

Reply:
Yes, I increased changed to 1.1.0 previously and am reverting that change.